docs/fivem-gamebuild.md

```diff
@@ -26,6 +26,8 @@ This is an overview of all available DLC and its Build IDs:
 | 2699     | The Criminal Enterprises  |
 | 2802     | Los Santos Drug Wars      |
 | 2944     | San Andreas Mercenaries   |
+| 3095     | The Chop Shop             |
+| 3258     | Bottom Dollar Bounties    |
```

Comment:
3258 doesn't seem to be officially documented yet is this actually a valid build?
https://docs.fivem.net/docs/server-manual/server-commands/#sv_enforcegamebuild-build

Reply:
The docs haven't been updated yet, but it's currently on canary. So this could either be merged now or when it hits release. https://github.com/citizenfx/fivem/commit/636ff8d70e9f8327d7be7956021c15969ac9f998

Reply:
I see, thanks for linking the commit!